src/content/docs/de/guides/integrations-guide/alpinejs.mdx

```diff
@@ -7,6 +7,7 @@ category: renderer
 i18nReady: true
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import Card from '~/components/Landing/Card.astro'
 
```

Comment:
Looks like you forgot to import the `Grid` component here

Reply:
fixed